packages/authentication/src/__tests__/unit/strategy-adapter.unit.ts

```diff
@@ -16,33 +16,37 @@ describe('Strategy Adapter', () => {
     it('calls the authenticate method of the strategy', async () => {
       let calledFlag = false;
       // TODO: (as suggested by @bajtos) use sinon spy
-      class Strategy extends MockStrategy {
+      class Strategy extends MockPassportStrategy {
```

Comment:
@jannyHou , Please use a custom strategy class other than `Strategy` so it is not confused by the `Strategy` class used by passport-strategy. thx.

Reply:
@jannyHou ^^^ Please see my comments. thx.

Reply:
@emonddr Thanks good catch. This code change is merged in PR https://github.com/strongloop/loopback-next/pull/2859. I can address it in the coming doc PR that moves https://github.com/strongloop/loopback-next/blob/master/packages/authentication/docs/authentication-system.md to the root level README.md file.